src/index.js

```diff
@@ -6,7 +6,7 @@ let win
 
 function createWindow () {
   // Create the browser window.
-  win = new BrowserWindow({width: 800, height: 600})
+  win = new BrowserWindow({width: 1280, height: 800})
```

Comment:
Good idea to set a motre reasonable default window size.
Maybe it would be even more useful to set the window maximized by default with `win.maximize()`.